.github/workflows/sycl-benchmark-aggregate.yml

```
@@ -0,0 +1,81 @@
+name: Aggregate benchmark averages
+
+on:
+  schedule:
+  - cron: 0 1 * * *
+  workflow_dispatch:
+    inputs:
+      cutoff_timestamp:
+        description: "Timestamp "
+        type: string
+        required: false
+  workflow_call:
+    inputs:
+      cutoff_timestamp:
+        type: string
+        required: false
+
+permissions:
+  contents: read
+
+jobs:
+  aggregate:
+    name: Aggregate average value for all metrics (median)
+    runs-on: ubuntu-latest
+    if: github.repository == 'intel/llvm'
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        path: llvm
+        sparse-checkout: |
+          devops/scripts/benchmarking
+    - name: Load benchmarking configuration
+      run: |
+        # TODO isolate sanitation from benchmark.sh different script
+        # use sanitation here
+        . llvm/devops/scripts/benchmarking/benchmark-ci.conf;
+        echo "PERF_RES_GIT_REPO=$PERF_RES_GIT_REPO" >> $GITHUB_ENV
+        echo "PERF_RES_BRANCH=$PERF_RES_BRANCH" >> $GITHUB_ENV
+        echo "PERF_RES_PATH=$PERF_RES_PATH" >> $GITHUB_ENV
+
+        if [ -z '${{ inputs.cutoff_timestamp }}' ]; then
+          # No time given, use default time period
+          echo "CUTOFF_TIMESTAMP=$(date --date="$AVERAGE_CUTOFF_RANGE" +"$TIMESTAMP_FORMAT")" >> $GITHUB_ENV
+        else
+          # If the provided time is a unix `date` timestamp, convert the time to our format
```

```suggestion
          # If the provided time is a unix `date` timestamp, convert the time to our format
```
What's the timestamp format we use? How and why is it different from Unix `date` timestamp?